.gitignore

```diff
@@ -1,2 +1,4 @@
 /vendor/
 /node_modules/
+composer.lock
+package-lock.json
```

Comment:
You don't necessarily need to remove the `package-lock.json` - any JS build would be specific to this repo still. It's only `composer.lock` files that are ignored in dependencies installed via composer